### PR TITLE
Make readme a bit more helpful

### PR DESCRIPTION
**Pull request by Qard**
Just making the readme a little more helpful for people newly landing on this repo. Any other info you think we should include for first-time viewers?

### CODE REVIEW

**Review by Mariatta — APPROVED** [review]
Overall looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-# Landing Page
+# Cambie Collective
 
-## Getting started
+Supporting resilient, inclusive tech and maker communities that empower positive change.
```
Comment (Mariatta) [inline README.md]: Should this mention "in Vancouver area" or something?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
 
-To get going locally, run:
-`npm run develop`
+## Event groups we support
+
+- [NodeSchool YVR](https://nodeschool.io/vancouver/)
+- [Code & Coffee YVR](https://www.meetup.com/codecoffeeyvr/)
+- [PyLadies Vancouver](https://www.meetup.com/PyLadies-Vancouver/)
+- [VanFP](https://www.meetup.com/Vancouver-Functional-Programmers/)
+- [VanJS](https://www.meetup.com/vancouver-javascript-developers/)
```
Comment (caffodian) [inline README.md:11]: I would just use the suggest changes feature but I don't actually know how to get to it.  can we add 
```
[VanPy](https://www.meetup.com/vanpyz/)
``` 
please?

(I got in touch with everyone before I left on vacation in December, and we actually have the group active again.  Yay!)

**Review by caffodian — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Qard) [reply README.md:11]: done.

File: README.md (hunk 1 above)
Comment (Qard) [reply README.md]: fixed

**Review by Mariatta — APPROVED** [review]

**Comment by Mariatta** [comment]
Thanks for working on this!

**Review by keywordnew — APPROVED** [review]
LGTM! Thank you for making a way more helpful README! 💛

**Comment by keywordnew** [comment]
Maybe a suggestion to open an issue if people want to help in non-code and code ways? We can track those and connect people with activities.

Of course sometimes, people are free to jump in directly! 💯 cc/ (@oikake)

**Comment by keywordnew** [comment]
Merging, y'all! 🎉

**Pull request merged**